src/main.routes.tsx

```diff
@@ -0,0 +1,18 @@
+import { Route } from 'react-router-dom';
+
+import { Authenticate } from '~/core/utils/routing';
+import FourOhFour from '~/features/404/public/FourOhFourView';
+import App from '~/features/app/public/AppView';
+import Login from '~/features/authentication/public/LoginView';
+
+const MainRoutes = (
+    <>
+        <Route element={<Login />} path="/login" />
```

Comment:
This just reminded me - those `Suspense(<Element/>)` wrappers I had here was what was allowing the chunking per route. More specifically, I guess, was them being wrapped in `React.Suspense`. I don't think its a concern now but we probably want to have some kind of go forward plan for those. I didn't love them being wrapped in that function but React Router is incredibly opinionated as to what a Route looks like so I'm not sure we'll have many different options.

Also, I suppose, we need to see if we can chunk the imports to a route and not just the route itself. Which is just me musing and not calling that out as actionable here.

Reply:
sounds good. yeah chunking is definitely something I'd like to have a concrete workable plan for from the get-go, as that was something we always wanted to do in Ember, but just never got around to it.